### PR TITLE
Implement comments management in Admin Panel

### PR DESCRIPTION
**Pull request by hedudelgado**
@alexnoox

### CODE REVIEW

File: frontend-admin/src/app/components/mnoe-api/admin/ratings.svc.coffee [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  # GET List /mnoe/jpi/v1/admin/app_user_ratings
+  @list = () ->
+    MnoeAdminApiSvc.all('app_user_ratings').getList().then(
+      (response) ->
+        response
+      (error) ->
+        # Display an error
+        $log.error('Error while fetching ratings', error)
+        toastr.error('An error occured while fetching the ratings.')
+    )
```
Comment (alexnoox) [inline frontend-admin/src/app/components/mnoe-api/admin/ratings.svc.coffee]: If you want to catch the errors, you can use `catch(errorCallback)`:
```coffeescript
# GET List /mnoe/jpi/v1/admin/app_user_ratings
@list = () ->
  MnoeAdminApiSvc.all('app_user_ratings').getList().catch(
    (error) ->
      # Log and display an error
      $log.error('Error while fetching ratings', error)
      toastr.error('An error occured while fetching the ratings.')
  )
```

File: frontend-admin/src/app/components/mnoe-api/admin/ratings.svc.coffee [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  # future implementation waiting for backend to be ready
+  # @removeRating = (id) ->
+  #   promise = MnoeAdminApiSvc.one('app_user_rating', id).then(
+  #     (response) ->
+  #       response
+  #     (error) ->
+  #       # Display an error
+  #       $log.error('Error while deleting rating', error)
+  #       toastr.error('An error occured while deleting the rating.')
+  #   )
```
Comment (alexnoox) [inline frontend-admin/src/app/components/mnoe-api/admin/ratings.svc.coffee]: Please remove dead/commented code

File: frontend-admin/src/app/components/mnoe-ratings-list/mnoe-ratings-list.coffee [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        (response) ->
+          # Remove the edit mode for this rating
+          delete scope.editmode[rating.id]
+      )
+
+    # Future implementation
+    # scope.remove = (rating) ->
+    #   MnoeRatings.removeRating(rating.id).then( ->
+    #     toastr.success("The #{rating.user_name}\'s rating has been successfully removed.")
+    #   )
```
Comment (alexnoox) [inline frontend-admin/src/app/components/mnoe-ratings-list/mnoe-ratings-list.coffee]: Please remove dead/commented code

File: frontend-admin/src/app/views/ratings/ratings.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<div class="row">
+  <div class="col-xs-8 col-lg-offset-1 top-buffer-1">
+  </div>
+</div>
```
Comment (alexnoox) [inline frontend-admin/src/app/views/ratings/ratings.html]: If there is nothing in this row, you can remove it.

**Comment by x4d3** [comment]
@alexnoox I am going to merge this in the feature branch, but I will address your comment and correct it in my next commit. 
You will be able to review everything when you merge the feature branch.

**Pull request merged**